examples/GettingStarted.java

```diff
@@ -9,34 +9,46 @@
 import com.gengo.client.GengoClient;
 import com.gengo.client.enums.Tier;
 import com.gengo.client.exceptions.GengoException;
+import com.gengo.client.payloads.Attachment;
 import com.gengo.client.payloads.TranslationJob;
 
 /* Basic runnable example */
 public class GettingStarted {
 
-	private GengoClient Gengo;
+    private GengoClient Gengo;
```

Comment:
Basically, lowercase is used for private variable, so, `private GengoClient gengo;` is better.